docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -0,0 +1,61 @@
+Referencing Content Blocks in Library V2
+--------------------------------------------------
+
+Status
+=======
+Pending
+
+Context
+=======
+Currently, the courses are being stored in the Modulestore and the libraries are stored in both Modulestore and the Blockstore. We want the library content to be stored in the Blockstore.
+We want to have a content reuse mechanism that allows authors to incorporate content (XBlocks) from content libraries into their courses without making a copy of it; in other words, we want a referencing mechanism for library content. 
+
+We are currently building out the functionality of libraries v2, and trying to migrate all usage of content libraries onto Blockstore.
+The benefits of using the Blockstore backed v2 libraries are:
```

Comment:
- `blockstore-backed`
- It'd be nice to use `content libraries` consistently over just `libraries`.

Reply:
True, I have tried to be consistent everywhere, feel free to point me to the place I might have missed :)